src/BuildScriptGeneratorCli/Commands/Binders/BuildCommandBaseBinder.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
```

Comment:
For all new files, please make sure that they have the Microsoft license header (you can pull this from any other `.cs` file in the repository)